features/admin.users.v1/constants/user-management-constants.ts

```diff
@@ -497,5 +497,15 @@ export const USER_ACCOUNT_STATUS_FILTER_OPTIONS: DropdownChild[] = [
         key: "PENDING_AP",
         text: "users:advancedSearch.accountStatusFilter.options.pendingInitialPasswordSetup",
         value: "urn:scim:wso2:schema:accountState eq PENDING_AP"
+    },
+    {
+        key: "PENDING_PRIMARY_EMAIL",
+        text: "users:advancedSearch.accountStatusFilter.options.pendingPrimaryEmailVerification",
+        value: "urn:scim:wso2:schema:emailVerified ne true"
```

Comment:
just to make sure, do we support ne operator now?

Reply:
With this effort we have introduced ne user filtering support